### PR TITLE
Database changes for Dressing Room 2

### PR DESCRIPTION
**Pull request by aanchal07**
The following **tables have been added** to database for Dressing Room 2 implementations:

1. Clothes
2. Hair
3. Accessories

These tables have the following **attributes** : ID, Name, Points, Purchased.

These files have been imported from the **csv files in the asset folder**.

Currently the points required to purchase a feature is **displayed below the image** of feature in the Dressing Room 2. 

**Portrait:**
db1

db2
 

**Landscape:**
db3

db4

### CODE REVIEW

**Comment by aanchal07** [comment]
@chhavip : Please review. :)

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 1]
```diff
@@ -11,6 +11,9 @@
 import android.widget.LinearLayout;
 import android.widget.TextView;
 import com.akexorcist.roundcornerprogressbar.IconRoundCornerProgressBar;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 2]
```diff
@@ -11,6 +11,9 @@
...
+import junit.framework.TestCase;
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 3]
```diff
@@ -232,6 +243,14 @@ public void onClick(View v) {
             final ImageView imageViewGlasses = (ImageView) findViewById(R.id.imageViewGlasses);
             final ImageView imageViewHat = (ImageView) findViewById(R.id.imageViewHat);
             final ImageView imageViewNecklace = (ImageView) findViewById(R.id.imageViewNecklace);
+            final TextView tvHandbagPoints = (TextView) findViewById(R.id.tvHandbagPoints);
+            final TextView tvGlassesPoints = (TextView) findViewById(R.id.tvGlassesPoints);
+            final TextView tvHatPoints = (TextView) findViewById(R.id.tvHatPoints);
+            final TextView tvNecklacePoints = (TextView) findViewById(R.id.tvNecklacePoints);
+            tvHandbagPoints.setText("5");
+            tvGlassesPoints.setText("8");
+            tvHatPoints.setText("8");
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: remove hardocoded integers

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 4]
```diff
@@ -249,6 +268,7 @@ public void onClick(View v) {
                         photoNameField = ourRID.getClass().getField(eyeImageName);
                         bagView.setImageResource(photoNameField.getInt(ourRID));
                         imageViewhandbag.setImageResource(photoNameField.getInt(ourRID));
+                        tvHandbagPoints.setText(getmDbHandler().getPointsAccessories(bag) + "");
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: convert to string using one of the in built functions

**Comment by aanchal07** [comment]
@chhavip : Thanks for the review. Have done the changes as you suggested.

**Pull request merged**